src/ResourceManager/AnalysisServices/Commands.AnalysisServices.Test/ScenarioTests/AsTests.ps1

```diff
@@ -1,6 +1,11 @@
 ﻿<#
 .SYNOPSIS
 Tests Analysis Services server lifecycle (Create, Update, Get, List, Delete).
+In order to run this test successfully, Following environment variables need to be set.
+ASAZURE_TEST_ADMUSERS e.x. value 'asengsys@microsoft.com'
```

Comment:
the env is set at local testing VM. Currently no idea how to get rid of them, since we have some credentials and SAS keys we don't want to hard coded in the source code.